Dockerfiles/Dockerfile-tools

```
@@ -5,12 +5,17 @@ ARG VERSION
 # --------------------------------------------------------------------------------------------------
 # See ./builder for this image
 FROM cytopia/ansible-builder as builder
+ARG VERSION
 
 # Python packages (copied to final image)
 RUN set -eux \
-	&& pip3 install --no-cache-dir --no-compile \
-		dnspython \
-		mitogen \
+	&& if [ "${VERSION}" = "latest" ]; then \
+		pip3 install --no-cache-dir --no-compile dnspython mitogen; \
+	elif [ "${VERSION}" = "2.9" ]; then \
+		pip3 install --no-cache-dir --no-compile dnspython mitogen==0.2.10; \
+	else \
+		pip3 install --no-cache-dir --no-compile dnspython mitogen; \
+	fi \
```

I suppose it can be simplified as follows:
```suggestion
	&& if [ "${VERSION}" = "2.9" ]; then \
		pip3 install --no-cache-dir --no-compile dnspython mitogen==0.2.10; \
	else \
		pip3 install --no-cache-dir --no-compile dnspython mitogen; \
	fi \
```